README.md

```diff
@@ -1,8 +1,12 @@
 OME production services playbooks
 =================================
 
-**NOTE: We are in the process of migration of the playbooks in this repo to Rocky Linux 9/RHEL 9 OS but at the moment, they function only on CentOS 7 !!**
 These playbooks encapsulate the running of various production servers run by the OME team.
+
+At the moment, only the `ome-demoserver.yml` playbook is available here. This is a playbook for OMERO.demo server on https://demo.openmicroscopy.org OMERO.server and OMERO.web. You can read more about this [OMERO.demo server on our website](https://www.openmicroscopy.org/explore/).
+
+We are in the process of adding more OME team's production playbooks here.
```

Comment:
Is the intent of this repository to only hold OMERO production playbooks moving forward?

Reply:
> Is the intent of this repository to only hold OMERO production playbooks moving forward?

That was my initial intent when planning this PR and discussing the tactics previously with @jburel .
What else would you like to hold here @sbesson ?
The idea is to ``minimize and simplify`` wherever possible.
But happy to discuss about better strategies.

Reply:
I don't have anything different to propose. I was mostly enquiring as I saw the `omero` subfolder was maintained and this was historically used for separating the OMERO playbooks from the others.

Reply:
Yes, ``omero`` folder is maintained to get out of the conandrum of molecule not finding the roles folder. Maybe I could rename the folder to ``playbooks`` ?